codecarbon/data/hardware/cpu_power.csv

```diff
@@ -0,0 +1,2144 @@
+Name,TDP
```

Comment:
The file needs to be declared in `setup.py` `package_data` or it won't be packaged